### PR TITLE
Implement partition metadata binding

### PR DESCRIPTION
**Pull request by morenol**
This is a continuation of #1, it implements PartitionMetadata

### CODE REVIEW

File: src/partition.rs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    #[wasm_bindgen(method, getter = logEndOffset)]
+    pub fn log_end_offset(&self) -> u32 {
+        self.inner.status.leader.leo as u32
+    }
+
+    #[wasm_bindgen(method, getter = liveReplicas)]
+    pub fn live_replicas(&self) -> u32 {
+        self.inner.status.lsr
```
Comment (morenol) [inline src/partition.rs:31]: This looks like a typo in fluvio, right?

Comment (simlay) [reply src/partition.rs:31]: I'm not sure what the `lsr` stuff is. But it looks like you could use [`self.inner.status.lsr()`](https://github.com/infinyon/fluvio/blob/eff4820d437f0961352c4a287c6b93c89517993f/src/controlplane-metadata/src/partition/status.rs#L86-L88)

Comment (morenol) [reply src/partition.rs:31]: Sure, according to [this](https://github.com/infinyon/fluvio/blob/e72b67bac332d4060e2eb01de3923094a0b3db71/k8-util/helm/fluvio-sys/templates/crd_partition.yaml#L57) it means Live Replicas

**Review by simlay — APPROVED** [review]
Seems fine.

**Pull request merged**